### PR TITLE
Ft/gc/about

### PR DESCRIPTION
**Pull request by GraceChiamaka**
### What will this PR do
This PR would add the about component

### CODE REVIEW

**Comment by GraceChiamaka** [comment]
@majordwarf please review

**Comment by Cervator** [comment]
Thanks @GraceChiamaka :-)

Would you mind expanding a bit on your description? I take it this is drawing in content from the existing site repo powering terasology.org ? Have you made any particular modifications to make stuff work well? Have you tested the updates and if so do you have either a preview URL or a screenshot or two? That'll all help 👍

File: src/components/About/About.jsx [hunk 1]
```diff
@@ -1,8 +1,94 @@
...
+          <div className="row mt-6e">
+            <div className="col-md-7">
+              <h3 className="mb-3  mr-4">Modding API</h3>
+              <p className="text-justify">
+                Terasology is fully open source and is licensed under the Apache 2.0 
+                license for code and under CC BY 4.0 for artwork (see credits for some minor exceptions).
+                The Terasology project was and is becoming a stable platform for various types of gameplay settings in a voxel world. 
+                The creators and maintainers are a diverse mix of software developers, designers, game testers, 
+                graphic artists, and musicians. We encourage others to join
+              </p>
```
Comment (majordwarf) [inline src/components/About/About.jsx]: Same as above the content doesn't highlight any Modding API stuff, also no need to go technical here. Better change the topic to `Modular` rather than `Modding API` and edit the content accordingly.

File: src/components/About/About.jsx [hunk 2]
```diff
@@ -1,8 +1,94 @@
...
+            <div className="col-md-5 pt-5">
+            <div className="mod_bg"></div>
+            </div>
+            <div className="col-md-7">
+              <h3 className="mb-3 text-right mr-4">History</h3>
+              <p className="text-justify">
+              From the ground up, Terasology was built to be a super hackable and modular game. 
+              We host a large number of modules under the Terasology organization and many more which 
+              are maintained by individual enthusiasts. We welcome new ideas, both crazy and well thought-out 
+              for modules and game extensions from anyone and everyone, so feel free to talk to us on our <a className="text-success font-weight-bold" href="https://discordapp.com/invite/Terasology">Discord</a> or IRC (#terasology on Freenode).
```
Comment (majordwarf) [inline src/components/About/About.jsx]: The following content doesn't go along with the History header, better have something actually of how Terasology was born, the following [wiki page](https://github.com/MovingBlocks/Terasology/wiki/What-is-Terasology) might help you.

File: src/components/About/About.jsx [hunk 3]
```diff
@@ -1,8 +1,94 @@
...
+            The creators and maintainers are a diverse mix of software developers, designers, game testers, 
+            graphic artists, and musicians. We encourage others to join
+          </p>
+        </div>
+      </div>
+      <div className="row justify-content-center">
+        <div className="col-md-8">
+          <div className="row mt-6e">
+            <div className="col-md-5 pt-5">
+            <div className="mod_bg"></div>
```
Comment (majordwarf) [inline src/components/About/About.jsx]: I see you're using CSS background properties for the images to be displayed. It'd be better if we use `Img` component from `gatsby-image` package for better image processing and accessibility. Also I saw you are using Minecraft images/screenshots besides the information. It'd be better if you switch the images accordingly and remove the minecraft images. Same goes for other two images linked. 

Also I tired resizing the screen and it had problem when the screen size fell below 768px, switching to `Img` tag along with some media queries might help. Have a look at the `templates/post.jsx` to see how currently we are handling images.

File: src/components/About/About.css [hunk 4]
```diff
@@ -4,4 +4,57 @@
...
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.mod_group{
+  width: 100%;
+  height: 100%;
+  background-image: url('./images/community.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```
Comment (majordwarf) [inline src/components/About/About.css]: It's a good practice to add a newline at the end of the file More about it [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).

TL;DR - Just add an empty line at end of the file.

File: src/components/About/About.css [hunk 5]
```diff
@@ -4,4 +4,57 @@
...
+  flex-direction:column;
 }
+
+.about_bg{
+  background-image: url('./images/ss_2.jpg');
+  width: 100%;
+  height: 400px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: right;
```
Comment (majordwarf) [inline src/components/About/About.css:16]:
```suggestion
  background-position: right;
  border-radius: 5px;
```

Minor edit to maintain a normalized styling throughout the website.

File: src/components/About/About.css [hunk 6]
```diff
@@ -4,4 +4,57 @@
...
+}
+
+.about__bg-overlay{
+  width: 100%;
+  height: 400px;
+  background: rgba(40, 167, 69, 0.6);
+  color: #fff;
+  text-align: center;
+  padding-top: 2rem;
```
Comment (majordwarf) [inline src/components/About/About.css:26]:
```suggestion
  padding-top: 2rem;
  border-radius: inherit;
```
Follow up from the above message.

**Review by majordwarf — CHANGES_REQUESTED** [review]
On a sidenote, please use `reactstrap` that will help avoid writing `className` attribute again and again, providing a much better readability and only focuses on the user-defined classes.

**Comment by GraceChiamaka** [comment]
@majordwarf All your comments have been noted and will be implemented
Thanks

**Comment by majordwarf** [comment]
@GraceChiamaka Are you still working on the issue?

**Comment by GraceChiamaka** [comment]
@majordwarf Yes I am.

**Comment by majordwarf** [comment]
@GraceChiamaka The images still needs to be changed in the History, Modding API and Community section, they still show minecraft images.

**Comment by GraceChiamaka** [comment]
@majordwarf Yes I am still working on changing those using gatsby-image

**Comment by GraceChiamaka** [comment]
@majordwarf I've attached screenshots of what the page looks like on desktop and mobile view here
Thanks
Screenhot_1 The Game
Screenshot_2 mobile

**Review by majordwarf — APPROVED** [review]
Great! Thanks for the PR and implementing the changes requested.
